frontend/src/utils/CampUtils.ts

```diff
@@ -138,6 +141,49 @@ export const getFormattedCampDateRange = (
   return "";
 };
 
+const getWeekDayKeyFromDateNum = (num: number): string => {
+  switch (num) {
+    case 0:
+      return "Su";
+    case 1:
+      return "Mo";
+    case 2:
+      return "Tu";
+    case 3:
+      return "We";
+    case 4:
+      return "Th";
+    case 5:
+      return "Fr";
+    default:
+      return "Sa";
+  }
+};
+
+// getSelectedWeekDaysFromDates takes an array of Dates and returns map of weekdays,
+// with value of true if camp session occurs on this weekday, otherwise false.
+// All camp sessions occur over a single week period.
+export const getSelectedWeekDaysFromDates = (
+  sessionDates: Date[],
+): Map<string, boolean> => {
+  const campDays = new Map<string, boolean>([
+    ["Su", false],
+    ["Mo", false],
+    ["Tu", false],
+    ["We", false],
+    ["Th", false],
+    ["Fr", false],
+    ["Sa", false],
+  ]);
+
+  sessionDates.forEach((date) => {
+    const key = getWeekDayKeyFromDateNum(date.getDay());
+    campDays.set(key, true);
```

Comment:
this should be value right

Reply:
I am using this function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/set

this essentially sets the day to be true if it has a workshop scheduled on that day (hence true being hardcoded)